templates/github_folder.groovy

```diff
@@ -0,0 +1,108 @@
+// Adapted from https://github.com/beedemo/pse-cje-sa/quickstart/init_22_github_org_project.groovy
```

Comment:
This URL didn't work for me; https://github.com/beedemo/pse-cje-sa/blob/5e5f07529d074153440c9ff0905dd0b0889d9bfa/quickstart/init_22_github_org_project.groovy does, however.